solver/src/main/java/org/chocosolver/solver/constraints/real/IbexHandler.java

```diff
@@ -125,7 +132,14 @@ void contract(RealPropagator prop) throws ContradictionException {
             filter = false;
             extractDomains();
             reif = getReif(prop);
-            int result = mIbex.contract(ibexCtr.get(prop), domains, reif);
+            int result;
+            if (contractionRatio == Ibex.RATIO) {
+                // Compatibility with ibex version previous to 2.8.8
+                result = mIbex.contract(ibexCtr.get(prop), domains, reif);
```

Comment:
I suppose a clear message in both the Javadoc and CHANGES.md is the least we can do.

Reply:
Done!